web/test/pages/profile.test.tsx

```diff
@@ -2026,4 +2056,20 @@ describe("profile", () => {
     ).toBeInTheDocument();
     expect(screen.getByTestId("snackbar-alert-ERROR")).toBeInTheDocument();
   };
+
+  const randomTradeNameLegalStructure = () => {
+    const tradeNameLegalStructures = LegalStructures.filter((x) => {
+      return x.hasTradeName;
+    });
+    const randomIndex = Math.floor(Math.random() * tradeNameLegalStructures.length);
+    return tradeNameLegalStructures[randomIndex].id;
+  };
+
+  const randomPublicFilingLegalStructure = () => {
+    const nonTradeNameLegalStructures = LegalStructures.filter((x) => {
+      return x.requiresPublicFiling;
+    });
+    const randomIndex = Math.floor(Math.random() * nonTradeNameLegalStructures.length);
+    return nonTradeNameLegalStructures[randomIndex].id;
+  };
```

Comment:
Same comment as above - we should be able to import the existing helper functions here instead of recreating